_translations/en.yml

```diff
@@ -750,7 +750,7 @@ en:
     address: "Address"
     addresstxt: "A Bitcoin address is <b>similar to a physical address or an email</b>. It is the only information you need to provide for someone to pay you with Bitcoin. An important difference, however, is that each address should only be used for a single transaction."
     bit: "Bit"
-    bittxt: "Bit is a common unit used to designate a sub-unit of a bitcoin - 1,000,000 bits is equal to 1 bitcoin (BTC or B⃦). This unit is usually more convenient for pricing tips, goods and services."
+    bittxt: "Bit is a common unit used to designate a sub-unit of a bitcoin - 1,000,000 bits is equal to 1 bitcoin (BTC or B?). This unit is usually more convenient for pricing tips, goods and services."
```

Comment:
NACK this change.  This should be in a separate PR, but even if it was, it doesn't render correctly here:

![screenshot-github com 2015-08-28 07-08-44](https://cloud.githubusercontent.com/assets/61096/9545035/ddc0742e-4d53-11e5-8218-b07633f9657a.png)